scipy/stats/tests/test_discrete_distns.py

```diff
@@ -1,6 +1,6 @@
-from scipy.stats import betabinom, hypergeom, bernoulli, boltzmann
+from scipy.stats import betabinom, hypergeom, nhypergeom, bernoulli, boltzmann
 import numpy as np
-from numpy.testing import assert_almost_equal, assert_equal, assert_allclose
+from numpy.testing import assert_almost_equal, assert_equal, assert_allclose, suppress_warnings
```

Comment:
I don't think this import of `suppress_warnings` will be necessary, but if it is, this line needs to be broken after `assert_allclose,` to keep the maximum line length at 79 characters or fewer (PEP 8).